tests/test_freshdesk_pagination.py

```diff
@@ -1,58 +1,89 @@
-from tap_tester import menagerie, connections, runner
-import re
+from math import ceil
+from tap_tester import connections, runner, LOGGER
 
 from base import FreshdeskBaseTest
 
 class PaginationTest(FreshdeskBaseTest):
 
     def name(self):
-        return "tap_freshdesk_pagination_test"
+        return "tap_tester_freshdesk_pagination_test"
 
     def test_name(self):
-        print("Pagination Test for tap-freshdesk")
+        LOGGER.info("Pagination Test for tap-freshdesk")
 
     def test_run(self):
+        """
+        • Verify that for each stream you can get multiple pages of data.  
+        This requires we ensure more than 1 page of data exists at all times for any given stream.
+        • Verify by pks that the data replicated matches the data we expect.
+        """
+        
+        # For roles stream data present in test account is limited. So, adding configurable page_size "2" 
+        streams_to_test_1 = {"roles"}
+        self.run_test(streams_to_test_1, 2)
 
-        # instantiate connection
-        conn_id = connections.ensure_connection(self)
-
-        # Add supported streams 1 by 1
-        streams_to_test = {'agents', 'tickets'}
-
-        # Run check mode
-        # Check mode has no catalog discovery for freshdesk
-        check_job_name = self.run_and_verify_check_mode(conn_id)
+        streams_to_test_2 = self.expected_streams() - streams_to_test_1
+        self.run_test(streams_to_test_2, 100)
 
-        # Run sync mode
-        sync_record_count = self.run_and_verify_sync(conn_id)
-        sync_records = runner.get_records_from_target_output()
+    def run_test(self, streams_to_test ,page_size):
 
-        # Test by stream
-        for stream in streams_to_test:
-            with self.subTest(stream=stream):
+        # Page size for pagination supported streams
+        self.PAGE_SIZE = page_size
 
-                record_count = sync_record_count.get(stream, 0)
+        # Instantiate connection
+        conn_id = connections.ensure_connection(self)
 
-                sync_messages = sync_records.get(stream, {'messages': []}).get('messages')
+        # To collect "time_entries", "satisfaction_ratings" pro account is needed. Skipping them for now.
+        expected_streams = streams_to_test
```

Comment:
Update comment in all tests. As not excluding here, no need to mention it.

Reply:
Removed